packages/cashier/src/components/cashier-locked/__tests__/cashier-locked.spec.tsx

```diff
@@ -19,31 +19,6 @@ describe('<CashierLocked />', () => {
         mockUseCashierLocked.mockReturnValue(false);
     });
 
-    it('should show the proper message if there is a crypto cashier maintenance', () => {
-        const mock_root_store = mockStore({
-            client: {
-                account_status: { cashier_validation: ['system_maintenance'] },
-                current_currency_type: 'crypto',
-                mt5_login_list: [
-                    {
-                        account_type: 'demo',
-                        sub_account_type: 'financial_stp',
-                    },
-                ],
-            },
-        });
-
-        render(<CashierLocked />, {
-            wrapper: ({ children }) => <CashierProviders store={mock_root_store}>{children}</CashierProviders>,
-        });
-
-        expect(
-            screen.getByText(
-                'Our cryptocurrency cashier is temporarily down due to system maintenance. You can access the Cashier in a few minutes when the maintenance is complete.'
-            )
-        ).toBeInTheDocument();
-    });
-
```

Comment:
can we still keep this but with the new message for crypto accounts?

Reply:
but we don't have condition to check wether its crypto or not anymore. So, i just think its not necessary to include that test case 🙏